src/foam/u2/view/ArrayView.js

```diff
@@ -41,6 +41,24 @@ foam.CLASS({
       name: 'enableRemoving',
       value: true
     },
+    {
+      class: 'Boolean',
+      name: 'allowDuplicates',
+      defaultValue: true
+    },
+    {
+      name: 'disabledData_',
+      documentation: 'Optional list of choices that should be disabled',
+      factory: function() {
+        return [];
+      }
```

Comment:
Added this place holder (which i link to actual data property if allowDuplicates = false)
I only added this as I was not able to pass empty slot to ReferenceView if disabledData should be ignored

Reply:
Updated to use expression per Mann's comment